docs/cmd/kn_plugin_list.md

```diff
@@ -0,0 +1,34 @@
+## kn plugin list
+
+list all visible plugin executables on a user's PATH
+
+### Synopsis
+
+List all available plugin files on a user's PATH.
+
+		Available plugin files are those that are:
+		- executable
+		- anywhere on the user's PATH
+		- begin with "kn-
+
+```
+kn plugin list [flags]
+```
+
+### Options
+
+```
+  -h, --help        help for list
+      --name-only   If true, display only the binary name of each plugin, rather than its full path
```

Comment:
I think this should be reversed. In most cases I don't think people will care where the exe lives and when they do care it's probably because they're debugging to see why things are not acting correctly.

Reply:
OK, up for debate. Especially if we decide to have a fix location.